webcompat/static/js/lib/comments.js

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-var issues = issues || {};
+var issues = issues || {}; // eslint-disable-line no-use-before-define
```

Comment:
One day we need to really move away from these globals and use some kind of module system... I blame myself. ;_;